classes/announcement/Collector.php

```diff
@@ -27,13 +27,20 @@
  */
 class Collector implements CollectorInterface
 {
+    public const ORDERBY_DATE_POSTED = 'date_posted';
+    public const ORDERBY_DATE_EXPIRE = 'date_expire';
+    public const ORDER_DIR_ASC = 'ASC';
+    public const ORDER_DIR_DESC = 'DESC';
+
     public DAO $dao;
     public ?array $contextIds = null;
     public ?string $isActive = null;
     public ?string $searchPhrase = null;
     public ?array $typeIds = null;
     public ?int $count = null;
     public ?int $offset = null;
+    public string $orderBy = self::ORDERBY_DATE_POSTED;
```

Comment:
This looks like it'll implicitly solve the sorting issue filed here: https://github.com/pkp/pkp-lib/issues/9226